mtags/src/main/scala/scala/meta/internal/pc/HoverMarkup.scala

```diff
@@ -17,9 +17,7 @@ object HoverMarkup {
       forceExpressionType: Boolean = false
   ): String = {
     val markdown = new StringBuilder()
-    val needsExpressionType =
```

Comment:
This caused the changes in HoverDocSuite, but I don't think we should show expression in that case (it also causes some issues with Scala 3, which are currently hard to deal with). We should most likely fix the types in for `emptyList`.